### DISCUSSION TITLE
Can anyone help with the mDNS reflector install?

### DISCUSSION THREAD

**Discussion by YargaBHK** [unifi-utilities/unifi-common#230] [Q&A] [answered]
I tried to run even the first step--install the Debian package and the website where it is located is gone.  

Does anyone have a step by step instruction how to install this?  Would really appreciate it.

Thanks!

**Comment by YargaBHK** [comment]
Here is the error I get once I point to the new location of this Debian package


dpkg-deb: error: 'udm-boot_1.0.5_all.deb' is not a debian format archive

**Reply by boostchicken** [reply]
THis is fixed.  It was my bad

**Comment by boostchicken** [comment, marked as answer]
This is fixed, forgot to pay my AWS bill ;)
